client/tailscale/localclient.go

```diff
@@ -269,8 +269,9 @@ func (lc *LocalClient) Profile(ctx context.Context, pprofType string, sec int) (
 }
 
 // BugReport logs and returns a log marker that can be shared by the user with support.
-func (lc *LocalClient) BugReport(ctx context.Context, note string) (string, error) {
-	body, err := lc.send(ctx, "POST", "/localapi/v0/bugreport?note="+url.QueryEscape(note), 200, nil)
+func (lc *LocalClient) BugReport(ctx context.Context, note string, diagnose bool) (string, error) {
```

Comment:
per chat: keep this method as-is and add a new BugReportOpt or something that takes a struct of things

and document diagnose bool